scripts/launch/demo-cluster/default.nix

```diff
@@ -103,8 +106,7 @@ in pkgs.writeScript "demo-cluster" ''
   SYNCED=0
   while [[ $SYNCED != 100 ]]
   do
-    # TODO: switch to https when wallet-debug is removed
-    PERC=$(curl --silent -k http://localhost:8090/api/v1/node-info | jq .data.syncProgress.quantity)
+    PERC=$(curl --silent -k --cert ${stateDir}/tls-files/client.pem https://localhost:8090/api/v1/node-info | jq .data.syncProgress.quantity)
```

Comment:
why `-k` here?

Reply:
Good question. I honestly just added the `--cert` and preserved what was already there. 
Nevertheless, we do have the `ca.crt` however at that stage as well so we could use it all-the-same.